land_use/models.py

```diff
@@ -0,0 +1,247 @@
+        _('Area agricultural polygon ha'),
+        null=True,
+        blank=True,
+    )
+    nu_area_cr_ha = models.FloatField(
+        _('Area clear cut polygon ha'),
+        null=True,
+        blank=True,
+    )
+    nu_area_dg_ha = models.FloatField(
+        _('Area degradation polygon ha'),
+        null=True,
+        blank=True,
+    )
+    nu_area_ma_ha = models.FloatField(
+        _('Area body of water polygon ha'),
+        null=True,
+        blank=True,
+    )
+    nu_area_no_ha = models.FloatField(
+        _('Area not observed polygon ha'),
+        null=True,
+        blank=True,
+    )
+    nu_area_rv_ha = models.FloatField(
+        _('Area highway polygon ha'),
+        null=True,
+        blank=True,
+    )
+    nu_area_sv_ha = models.FloatField(
+        _('Area forestry polygon ha'),
+        null=True,
+        blank=True,
+    )
+    nu_area_vi_ha = models.FloatField(
+        _('Area village polygon ha'),
+        null=True,
+        blank=True,
+    )
+    nu_area_vn_ha = models.FloatField(
+        _('Area natural vegetation polygon ha'),
+        null=True,
+        blank=True,
+    )
+    nu_area_mi_ha = models.FloatField(
+        _('Area mining polygon ha'),
+        null=True,
+        blank=True,
+    )
+    nu_area_ha = models.FloatField(
+        _('Area polygon ha'),
+        null=True,
+        blank=True,
+    )
+    nu_area_km2 = models.FloatField(
+        _('Area polygon km2'),
+        null=True,
+        blank=True,
+    )
+    geom = models.GeometryField(
+        _('Geometry Field'),
+        srid=4326,
+        null=True,
+        blank=True,
+    )
+
+    class Meta:
+        """Meta class for `models.LandUseTI` model."""
+        app_label = 'land_use'
+        verbose_name = 'Land Use Mapping TI'
+        verbose_name_plural = 'Land Use Mapping TIs'
+        # db_table = 'funaidados\".\"img_analise_consolidado_oneatlas_dissolvido_por_ti_a'
+        # managed = False
+
+
+class LandUseClasses(models.Model):
+    """LandUseClasses model data for land_use model."""
+    id = models.IntegerField(
+        _('primary key'),
+        unique=True,
+        primary_key=True,
+    ),
+    sg_uf = models.CharField(
+        _('Brazilian states flag'),
+        max_length=255,
+        null=True,
+        blank=True
+    )
+    no_ti = models.CharField(
+        _('Name of Indigenous Lands'),
+        max_length=255,
+        null=True,
+        blank=True
+    )
+    co_funai = models.IntegerField(
+        _('Funai code for Indigenous Land'),
+        null=True,
+        blank=True
+    )
+    dt_homologada = models.CharField(
+        _('Data of ratification of Indigenaous Lands'),
+        max_length=255,
+        null=True,
+        blank=True
+    )
+    ds_cr = models.CharField(
+        _('Regional Coordenation name'),
+        max_length=255,
+        null=True,
+        blank=True
+    )
+    co_cr = models.BigIntegerField(
+        _('Regional Coordenation code'),
```

Comment:
Coordination

Reply:
CR code is big int? Are you sure about it?

Reply:
Yes, this is a numeric code defined by funai that has 11 characters of size.
We adopted BigIntegerField type throughout the project.